lib/components/pwa-prompt.tsx

```diff
@@ -98,16 +70,7 @@ export const PwaPrompt = ({
     ]
   );
 
-  const aboveMinVisits = iosPwaPrompt.visits >= promptOnVisit;
-  const belowMaxVisits = iosPwaPrompt.visits < promptOnVisit + timesToShow;
-
-  if (
-    (!iosPwaPrompt.isiOS ||
-      !aboveMinVisits ||
-      !belowMaxVisits ||
-      isDismissed) &&
-    !debug
-  ) {
+  if ((!shouldShowPrompt || isDismissed) && !isOpen) {
```

Comment:
Probably want to do something here so the prompt does not show if `isOpen` is actually false.